scripts/run-once-scripts/migrate-configuration-dynamo-to-ssm.ts

```diff
@@ -0,0 +1,48 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb';
```

Comment:
Minor, but if it doesn't affect anything much, can we move this into the run-once-scripts/postgres directory? If this involves re-testing anything that might point to it, though, it's not worth it.